src/features/users.ts

```diff
@@ -0,0 +1,22 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+import { getUsers } from '../api/users';
+
+export const loadUsers = createAsyncThunk('users/fetch', async () => {
+  const result = await getUsers().then(usersFromServer => usersFromServer);
```

Comment:
You don't need to use `.then()` after `await`. The `await` keyword already waits for the promise to resolve and then assigns the resolved value to the variable. So you can simply write `const result = await getUsers();`